backend/models/User.go

```diff
@@ -0,0 +1,11 @@
+package models
+
+import "github.com/jinzhu/gorm"
+
+//User that is used to register/login
+type User struct {
+	gorm.Model
```

Comment:
If you don't want the CreatedAt, UpdatedAt and DeletedAt columns you may be able to remove this line (not tested). However, keeping it seems reasonable, especially for this User struct.

see https://github.com/jinzhu/gorm/blob/master/model.go